src/app/modal/modal.html

```diff
@@ -123,7 +123,7 @@
                         <div class="clearfix"></div>
                     </div>
                     <!-- RESUME UPLOAD -->
-                    <div class="form-field upload-container" data-ngf-select data-ng-model="modal.ApplyService.form.resumeInfo" data-ngf-max-size="modal.configuration.maxUploadSize" data-ngf-multiple="false" data-ngf-accept="applyForm.$submitted=false; modal.validateResume($file);"  data-ng-if="!modal.hasAttemptedLIApply">
+                    <div class="form-field upload-container needsclick" data-ngf-select data-ng-model="modal.ApplyService.form.resumeInfo" data-ngf-max-size="modal.configuration.maxUploadSize" data-ngf-multiple="false" data-ngf-accept="applyForm.$submitted=false; modal.validateResume($file);"  data-ng-if="!modal.hasAttemptedLIApply">
```

Comment:
Heya Brian, the needsclick is to remove the effect of ng-fastclick from the upload container.  I have noticed that the behavior was causing the iPhone to require a double-click before the prompt to upload a file would appear.